components/prize_pool/commons/prize_pool.lua

```diff
@@ -705,6 +765,62 @@ function Placement:_parseOpponentArgs(input, date)
 	return Opponent.resolve(opponentData, date)
 end
 
+function Placement:_getLpdbData()
+	local entries = {}
+	for _, opponent in ipairs(self.opponents) do
+		local participant, image, imageDark, players
+		local playerCount = 0
+
+		if opponent.opponentData.type == Opponent.team then
+			local teamTemplate = mw.ext.TeamTemplate.raw(opponent.opponentData.template)
+
+			participant = teamTemplate and teamTemplate.page or ''
+			if self.parent.options.resolveRedirect then
+				participant = mw.ext.TeamLiquidIntegration.resolve_redirect(participant)
+			end
+
+			image = teamTemplate.image
+			imageDark = teamTemplate.imagedark
+		elseif opponent.opponentData.type == Opponent.solo then
+			participant = Opponent.toName(opponent.opponentData)
+			local p1 = opponent.opponentData.players[1]
+			players = {p1 = p1.pageName, p1dn = p1.displayName, p1flag = p1.flag, p1team = p1.team}
+			playerCount = 1
+		else
+			participant = Opponent.toName(opponent.opponentData)
+		end
+
+		local prizeMoney = tonumber(opponent.prizeRewards[PRIZE_TYPE_USD .. 1] or self.prizeRewards[PRIZE_TYPE_USD .. 1]) or 0
+		local lpdbData = {
+			image = image,
+			imagedark = imageDark,
+			date = opponent.date,
+			participant = participant,
+			participantlink = Opponent.toName(opponent.opponentData),
+			participantflag = opponent.opponentData.flag,
+			participanttemplate = opponent.opponentData.template,
+			players = players,
+			placement = self.placeStart .. (self.placeStart ~= self.placeEnd and ('-' .. self.placeEnd) or ''),
+			prizemoney = prizeMoney,
+			individualprizemoney = prizeMoney / playerCount,
```

Comment:
will error for team opponents (division by 0)

Reply:
Division by 0 is allowed in lua (returns the number `NaN`), which is then stored as 0 in LPDB.
But can change it to be more explicit.

Reply:
oh so it uses the 1-point compactification
alright fine for me to use the compactification point then